File: src/main/java/io/github/boogiemonster1o1/cartses/client/MinecartWithGlowstoneEntityRenderer.java
package io.github.boogiemonster1o1.cartses.client;

import io.github.boogiemonster1o1.cartses.entity.cart.MinecartWithGlowstoneEntity;

import net.minecraft.block.Blocks;
import net.minecraft.client.render.entity.EntityRenderDispatcher;
import net.minecraft.client.render.entity.MinecartEntityRenderer;
import net.minecraft.util.math.BlockPos;

import net.fabricmc.api.EnvType;
import net.fabricmc.api.Environment;

@Environment(EnvType.CLIENT)
public class MinecartWithGlowstoneEntityRenderer extends MinecartEntityRenderer<MinecartWithGlowstoneEntity> {
	public MinecartWithGlowstoneEntityRenderer(EntityRenderDispatcher dispatcher) {
		super(dispatcher);
	}

	@Override
	protected int method_27950(MinecartWithGlowstoneEntity entity, BlockPos blockPos) {
		return Blocks.GLOWSTONE.getDefaultState().getLuminance();
	}

	@Override
	protected int getBlockLight(MinecartWithGlowstoneEntity entity, BlockPos blockPos) {
		return Blocks.GLOWSTONE.getDefaultState().getLuminance();
	}
}

File: src/main/scala/io/github/boogiemonster1o1/cartses/client/MinecartWithGlowstoneEntityRenderer.scala
package io.github.boogiemonster1o1.cartses.client

import io.github.boogiemonster1o1.cartses.entity.cart.MinecartWithGlowstoneEntity
import net.fabricmc.api.{EnvType, Environment}
import net.minecraft.block.Blocks
import net.minecraft.client.render.entity.{EntityRenderDispatcher, MinecartEntityRenderer}
import net.minecraft.util.math.BlockPos;


@Environment(EnvType.CLIENT) class MinecartWithGlowstoneEntityRenderer(val dispatcher: EntityRenderDispatcher) extends MinecartEntityRenderer[MinecartWithGlowstoneEntity](dispatcher) {

	override protected def method_27950(entity: MinecartWithGlowstoneEntity, blockPos: BlockPos): Int = Blocks.GLOWSTONE.getDefaultState.getLuminance

	override protected def getBlockLight(entity: MinecartWithGlowstoneEntity, blockPos: BlockPos): Int = Blocks.GLOWSTONE.getDefaultState.getLuminance
}
